espnet2/iterators/category_iter_factory.py

```diff
@@ -88,8 +86,6 @@ def build_iter(self, epoch: int, shuffle: bool = None) -> DataLoader:
             if self.sampler_args["num_batches"] is not None:
                 batches = batches[: self.sampler_args.num_batches]
 
-            bs_list = [len(batch) for batch in batches]
```

Comment:
Is this OK?
(just want to confirm)

Reply:
`bs_list` is no used at any part of the code, so this assignment is unused.